.github/workflows/gh-pages.yaml

```
@@ -28,6 +28,9 @@ jobs:
       - name: Build Statistics
         run: ninja html-stats html-profile-stats -j2
         working-directory: ./build
+      - name: Render Policies (Using control files)
+        run: ninja render-policies
```

```suggestion
        run: ninja render-policies -j2
```

This can be improved by using two parallel jobs